api/build/id_token.go

```diff
@@ -75,19 +75,12 @@ import (
 // GetIDToken represents the API handler to generate a id token.
 func GetIDToken(c *gin.Context) {
 	// capture middleware values
+	l := c.MustGet("logger").(*logrus.Entry)
 	b := build.Retrieve(c)
 	cl := claims.Retrieve(c)
 	ctx := c.Request.Context()
 
-	// update engine logger with API metadata
-	//
-	// https://pkg.go.dev/github.com/sirupsen/logrus?tab=doc#Entry.WithFields
-	logrus.WithFields(logrus.Fields{
-		"build":   b.GetNumber(),
-		"org":     b.GetRepo().GetOrg(),
-		"repo":    b.GetRepo().GetName(),
-		"subject": cl.Subject,
-	}).Infof("generating ID token for build %s/%d", b.GetRepo().GetFullName(), b.GetNumber())
+	l.Infof("generating ID token for build %s/%d", b.GetRepo().GetFullName(), b.GetNumber())
```

Comment:
btw not sure if it was intended but i really like anything token/user/auth related being `info`

Reply:
agree. i'll double check to see if that's consistent.

Reply:
some changes in https://github.com/go-vela/server/pull/1145/commits/e1c4e37bb01afccec49b65490641faa9917f86ae - i'm actually a little torn on it. we get failed/success states from the request logger already, this will be fairly noisy. thoughts?